go-controller/pkg/crd/egressfirewall/v1/types.go

```diff
@@ -0,0 +1,75 @@
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// EgressNetworkFirewallRuleType indicates whether an EgressNetworkFirewallRule allows or denies traffic
+type EgressFirewallRuleType string
+
+const (
+	EgressFirewallRuleAllow EgressFirewallRuleType = "Allow"
+	EgressFirewallRuleDeny  EgressFirewallRuleType = "Deny"
+)
+
+// +genclient
+// +genclient:noStatus
```

Comment:
I assume this flag is wrong now that you added an empty status? Or does it mean "there's a status but it's empty"?

Reply:
took out the noStatus tag and added empty status instead